tests/load/filesystem/conftest.py

```diff
@@ -0,0 +1,49 @@
+import os
```

Comment:
i think @rudolfix wanted to have this in a utils.py file

Reply:
Sorry about that! Fixing now

Reply:
Fixed in f137adfff6e693cabfe16dbaeead46de7b313927